charts/kube-state-metrics/values.yaml

```diff
@@ -54,7 +54,8 @@ service:
   annotations: {}
 
 ## Additional labels to add to all resources
-customLabels: {}
+customLabels:
```

Comment:
can you remove the formatting changes that are not necessary for your change?

Reply:
done 👍